data-raw/healthy-lives/behavioural-risk-factors/smoking.R

```diff
@@ -0,0 +1,23 @@
+# ---- Load packages ----
+library(tidyverse)
+library(statswalesr)
+
+# ---- Scrape data from stats wales ----
+#Source:https://statswales.gov.wales/Catalogue/National-Survey-for-Wales/Population-Health/Adult-Lifestyles
+df <- statswales_get_dataset("hlth5072")
+
+# ---- Clean data ----
+#Filtered dataset includes percentage of adults (age 16+) who are self-reported smokers, age standardised
+#For Year, data from 2021-22 & 2022-23 has been combined
+hl_smoking <- df |>
+  filter(str_starts(Variable_ItemName_ENG, "Smoker") &
```

Comment:
In instances where it isn't self explanatory from the code, could  you please explain with a comment after each filter what you are filtering for?
I.e. it is not clear what "Per" or "14" or "A" represents.